bin/generate_participant.py

```diff
@@ -1 +1,29 @@
-1
+from pathlib import Path
+Path.cwd()
```

Comment:
Formatting convention: put all the modules (from xxx import xxx and import xxx) to the top.
Insert two blank lines before code start.